### ISSUE TITLE
Arcane Cascade macro does not work with translations

### ISSUE THREAD

**Issue by Far2Casual** [MrVauxs/pf2e-jb2a-macros#43]
The problem is that the damage type is localized by the PF2 system, so when the macro checks on the type, it doesn't find anything.

I was able to make it work by changing the test of the damage type, for example by doing: 
```
[...]
switch (damageType) {
        case "son": seq.effect()
[...]
```
("son" being the french translation of "sonic"), and then the macro works again.

So would there be any way to test again the translated value in the system ?

**Comment by Far2Casual** [comment]
Okay I think I have a quite elegant solution. In the macro, replace:
`let damageType = args[1].rinsedName.replace("stance:arcanecascade", "").replace("(", "").replace(")", "").trim().toLowerCase()`
by this (which actually looks up for the structured data):
`let damageType = args[1].item.flags.pf2e.rulesSelections.stanceArcaneCascade`

**Comment by MrVauxs** [comment]
Implemented for 2.5.0
